modules/acquia_cms_headless/tests/src/Functional/HeadlessHybridModeTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Drupal\Tests\acquia_cms_headless\Functional;
+
+/**
+ * Tests for acquia_cms_headless Hybrid mode.
+ *
+ * @group acquia_cms_headless
+ * @group low_risk
+ */
+class HeadlessHybridModeTest extends HeadlessBrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'acquia_cms_toolbar',
+  ];
+
+  /**
+   * Test Headless Dashboard in hybrid mode.
+   */
+  public function testAcmsWizardMenu(): void {
+    $account = $this->createUser();
+    $account->addRole('administrator');
+    $account->save();
+    $this->drupalLogin($account);
+    $assert_session = $this->assertSession();
+    $session = $this->getSession();
+    $page = $session->getPage();
+
+    // Get Fieldset to fetch Headless Dashboard link.
+    $menuItem = $assert_session->elementExists('css', '.toolbar-icon-acquia-cms-headless-dashboard');
+    $this->assertEquals('Headless dashboard', $menuItem->getText());
+    $assert_session->statusCodeEquals(200);
+
+    // Test if Enable headless mode checkbox works.
+    $this->drupalGet('/admin/tour/dashboard');
+    $checkbox = $assert_session->elementExists('css', '#edit-headless-mode');
+    $this->assertNotEmpty($checkbox);
+    $checkbox->check();
+    $page->pressButton('Save');
+
+    // Test Headless Dashboard link is not available.
+    $toolbar = $assert_session->elementExists('css', '#toolbar-administration');
+    $tour_link = $assert_session->elementExists('named', ['link', 'Acquia CMS Wizard'], $toolbar);
```

Comment:
@deepakmishra2 any reason to not to make $tour_link variable as a camel case ?

cc: @vishalkhode1